html/elements/input/range.json

```diff
@@ -36,7 +36,7 @@
                 "version_added": "3.1"
               },
               "safari_ios": {
-                "version_added": "5.1"
+                "version_added": "2"
```

Comment:
Is Safari 3.1 above correct?

Reply:
Just researched this some more.  The Safari 3.1 is correct, however it appears that iOS support wasn't introduced until Safari iOS 5.  Fixing!